File: buse-booking-service/src/main/java/com/trivago/buse_booking_service/messaging/booking_to_payment/BookingCancelledEvent.java
package com.trivago.buse_booking_service.messaging.booking_to_payment;

public class BookingCancelledEvent {
    private String bookingId;
    private String userId;
    private Double amount;
    private String currency;
    private String reason;

    public BookingCancelledEvent() {
    }

    public BookingCancelledEvent(String bookingId, String userId, Double amount, String currency, String reason) {
        this.bookingId = bookingId;
        this.userId = userId;
        this.amount = amount;
        this.currency = currency;
        this.reason = reason;
    }

    // Getters and Setters
    public String getBookingId() {
        return bookingId;
    }

    public void setBookingId(String bookingId) {
        this.bookingId = bookingId;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public Double getAmount() {
        return amount;
    }

    public void setAmount(Double amount) {
        this.amount = amount;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public String getReason() {
        return reason;
    }

    public void setReason(String reason) {
        this.reason = reason;
    }
}

File: nigar-payment-service/Events/BookingCancelledEvent.cs
using System.Text.Json.Serialization;

namespace nigar_payment_service.Events;

 public class BookingCancelledEvent
{
    [JsonPropertyName("bookingId")]
    public string BookingId { get; set; } = default!;

    [JsonPropertyName("userId")]
    public string UserId { get; set; } = default!;

    [JsonPropertyName("amount")]
    public decimal Amount { get; set; }

    [JsonPropertyName("currency")]
    public string Currency { get; set; } = default!;

    [JsonPropertyName("reason")]
    public string Reason { get; set; } = default!;
}